### DISCUSSION TITLE
如果重连op主路由，大陆的IPV6无法连接超时，其国外方向IPV6走代理正常

### DISCUSSION THREAD

**Discussion by caserangle** [vernesong/OpenClash#4070] [Q&A]
我的问题在于如果这个设备在连路由时候重启了OP后，他的大陆IPV6和国外IPV6（梯子已支持V6）都是正常的，但一旦我断开了连接重新连，就无法连接国内IPV6地址，且其余连此路由没有断开的依旧可以连接大陆方向的IPV6。我现在目前解决只能要不开启绕过大陆的IPV6不走内核，要不就是手动重启OPENCLASH，想问下这是哪里出的问题，我的IPV6不同代理方式三种都切换过了。还是不行。dnsmasq是上游，OP的解析交给了mosdns且没让mosdns劫持，是op劫持dnsmasq交给mosdns。我也尝试重启mosdns，但均不行。只能调试OP这边才能生效。[
[openclash调试日志.txt](https://github.com/user-attachments/files/17177727/openclash.txt)
](url)

**Comment by vernesong** [comment]
![image](https://github.com/user-attachments/assets/2100c211-2eb4-494b-9832-f2074786a16a)

**Comment by caserangle** [comment]
![屏幕截图 2024-09-30 135157](https://github.com/user-attachments/assets/76b12a98-76fa-49f8-b63b-8ee078a3399f)
已把IP动态伪装打开了，重启了路由和openclash测试还是不行QAQ，感觉可能是固件那边的问题
